src/constants/redact-strings.constant.ts

```diff
@@ -0,0 +1,4 @@
+export const REDACT_STRINGS = [
+  { searchValue: process.env.GRAPH_AUTHENTICATION_CLIENT_ID, replaceValue: '[Redacted]' },
```

Comment:
Does the process.env load here without dotenv?

Reply:
dotenv load happens in configuration, before this constant is used. process.env is global variable, so single dotenv load is enough.

I could add dotenv load here, but maybe it's excessive.